web/templates/pages/edit_server.html

```diff
@@ -825,6 +826,33 @@
 														</table>
 													</td>
 												</tr>
+												<tr style="display: <?php if ((empty($v_backup_type)) || !in_array(trim($v_backup_type, "'"),array('rclone'))){ echo 'none';} else {echo 'block';} ?>" id="backup_rclone">
+													<td>
+														<table>
+															<tr>
+																<td class="vst-text">
+																	<?=_('Host');?>
+																</td>
+															</tr>
+															<tr>
+																<td>
+																	<input type="text" size="20" class="vst-input" name="v_rclone_host" value="<?=trim($v_rclone_host, "'")?>">
+																</td>
+															</tr>
```

Comment:
This is the improved style I've been migrating to, should look the same:
```suggestion
															<tr>
																<td class="vst-text">
																	<label for="v_rclone_host" class="form-label"><?=_('Host');?></label>
																	<input type="text" class="form-control" name="v_rclone_host" id="v_rclone_host" value="<?=trim($v_rclone_host, "'")?>">
																</td>
															</tr>
```